lib/utils/errors.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import "strings"
+
+// IsUseOfClosedNetworkError returns true if the specified error
+// indicates the use of closed network connection
+func IsUseOfClosedNetworkError(err error) bool {
+	if err == nil {
+		return false
+	}
+	return strings.Contains(err.Error(), "use of closed network connection")
```

Comment:
If we do need this function, we have a constant for this error message: https://github.com/gravitational/teleport/blob/v6.0.0-alpha.1/constants.go#L633.